src/dfx/src/lib/operations/cycles_ledger.rs

```diff
@@ -105,3 +108,64 @@ pub async fn transfer(
 
     Ok(block_index)
 }
+
+pub async fn send(
+    agent: &Agent,
+    to: Principal,
+    amount: u128,
+    created_at_time: u64,
+    memo: Option<u64>,
+    from_subaccount: Option<icrc1::account::Subaccount>,
+    cycles_ledger_canister_id: Principal,
+) -> DfxResult<BlockIndex> {
+    let canister = Canister::builder()
+        .with_agent(agent)
+        .with_canister_id(cycles_ledger_canister_id)
+        .build()?;
+
+    let retry_policy = ExponentialBackoff::default();
+    let block_index: BlockIndex = retry(retry_policy, || async {
+        let arg = cycles_ledger_types::send::SendArgs {
+            from_subaccount,
+            to,
+            created_at_time: Some(created_at_time),
+            memo: memo.map(|v| v.into()),
+            amount: Nat::from(amount),
+        };
+        match canister
+            .update(SEND_METHOD)
+            .with_arg(arg)
+            .build()
+            .map(|result: (Result<BlockIndex, SendError>,)| (result.0,))
+            .call_and_wait()
+            .await
+            .map(|(result,)| result)
+        {
+            Ok(Ok(block_index)) => Ok(block_index),
+            Ok(Err(SendError::Duplicate { duplicate_of })) => {
+                println!(
```

Comment:
Would `error!` be more appropriate?

Reply:
It very well might.  As-is, it's consistent with `ledger transfer`, but it's not great.

Reply:
See https://github.com/dfinity/sdk/pull/3420.  Once approved, I'll make the same change here.